src/view/Start.vue

```diff
@@ -0,0 +1,132 @@
+<template>
```

Comment:
Ni upprepar er här mycket med apiAnropen. Kanske ska en helper för fetch()?? Där man skriver in bara type route och queries. Så slipper man skriva in token varje gång?